src/main/java/service/impl/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,32 @@
+package service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+import service.FruitTransactionParser;
+
+public class FruitTransactionParserImpl implements FruitTransactionParser {
+    private static final int OPERATION_FIELD_INDEX = 0;
+    private static final int FRUIT_FIELD_INDEX = 1;
+    private static final int QUANTITY_FIELD_INDEX = 2;
+    private static final int FIRST_VALUE_LINE_INDEX = 1;
+
+    @Override
+    public List<FruitTransaction> toTransactions(String data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        String[] lines = data.split(System.lineSeparator());
+        for (int i = FIRST_VALUE_LINE_INDEX; i < lines.length; i++) {
+            transactions.add(parseTransaction(lines[i]));
+        }
+        return transactions;
```

Comment:
Let's try to use Stream API here

Reply:
Done, but is here any more clever solution how to filter the line with fields names from the stream?